.github/workflows/sync-version.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: pull-request
+      uses: repo-sync/pull-request@v2
+      with:
+        source_branch: ""                                 # If blank, default: triggered branch
+        destination_branch: "OpenFOAM7"                   
+        pr_title: "Synchronize ${{ github.ref }} with OpenFOAM7" # Title of pull request
+        pr_body: "*An automated PR*: additional branches to sync: OpenFOAM4, OpenFOAM6, OpenFAOMdev"              # Full markdown support, requires pr_title to be set
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I am not sure how tokens work. Would this be your token? We also have @precice-bot that we can use for this.

Reply:
This is a special token: https://docs.github.com/en/actions/reference/authentication-in-a-workflow (I also learned it today). I guess it's more convenient to stick to the github token.